README.md

```diff
@@ -55,9 +55,46 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
 ### Playbook
 
 ```yaml
+---
 - hosts: all
+  any_errors_fatal: true
   roles:
-    - cloudalchemy.alertmanager
+    - ansible-alertmanager
```

Comment:
Please revert this line. Proper role name is `cloudalchemy.alertmanager` and that's what should be used.